src/plugins/orangehrmDashboardPlugin/Api/EmployeeOnLeaveAPI.php

```diff
@@ -67,7 +70,10 @@ public function getAll(): EndpointResult
         return new EndpointCollectionResult(
             EmployeeOnLeaveModel::class,
             $empLeaveList,
-            new ParameterBag([CommonParams::PARAMETER_TOTAL => $employeeCount])
+            new ParameterBag([
+                CommonParams::PARAMETER_TOTAL => $employeeCount,
+                self::PARAMETER_LEAVE_PERIOD => $this->getLeavePeriodService()->getCurrentLeavePeriod()
```

Comment:
@ManulMax I've edited your API slightly to get leave period in FE. please review this and if required send a PR to this branch with any fixes

Reply:
After adding this, seems BE test not changed. That means @ManulMax you haven't tested the meta section of the response right?

Reply:
No I tested it.